packages/apps/job-launcher/server/src/modules/job/job.service.ts

```diff
@@ -146,11 +149,18 @@ export class JobService {
       throw new NotFoundException(ErrorJob.NotCreated);
     }
 
+    const rate = await this.currencyService.getRate(
```

Comment:
getRate is always called to be passed to savePayment, I think we can move getRate call insice saveParyment function

Reply:
Good idea, but some times we are using reverse rate as here. What do you think about this case?

Reply:
https://github.com/humanprotocol/human-protocol/pull/654/files#r1263866867

Reply:
would swap the params `from` and `to` work?

Reply:
> would swap the params from and to work?

Reply:
Actually not, because of coingecko api, it will return correct response when token id it's first parameter. For example: 
✅ https://api.coingecko.com/api/v3/simple/price?ids=human-protocol&vs_currencies=usd
❌ https://api.coingecko.com/api/v3/simple/price?ids=usd&vs_currencies=human-protocol

@portuu3

Reply:
Can we have an enumerable of currencies and another one of tokens, so when the getRate function is called, depending on whether 'from' is a currency or a token, we reverse it, would that make sense?

Reply:
Yes, I will implement it